cocoa/src/toga_cocoa/app.py

```diff
@@ -8,6 +8,7 @@
 
 import toga
 from toga.handlers import NativeHandler
+from toga.screen import Screen as InterfaceScreen
```

Comment:
General convention is that Interface/Impl type descriptors go at the end of class names, not the start (i.e., HandlerInterface, not InterfaceHandler).

Reply:
Done`